### PR TITLE
Fix typo in the comment

### PR DESCRIPTION
**Pull request by damarisg**
# Description

Change verions to versions. I added this change to the master because the issue does not have the 4.8.0 project. 

---

### CODE REVIEW

**Review by QU3B1M — DISMISSED** [review]
LGTM!

File: tests/system/provisioning/manager_agent/vars/configurations.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ agent1_id: "001"
 agent1_hostname: "wazuh-agent1"
 agent1_key: "1111111111111111111111111111111111111111111111111111111111111111"
 agent1_package: "https://packages.wazuh.com/4.x/apt/pool/main/w/wazuh-agent/wazuh-agent_4.1.5-1_amd64.deb"
-agent1_user: "ossec" #Only valid for agents verions<4.2.5
+agent1_user: "ossec" #Only valid for agents versions<4.2.5
```
Comment (davidjiglesias) [inline tests/system/provisioning/manager_agent/vars/configurations.yml:14]:
```suggestion
agent1_user: "ossec" #Only valid for agents version<4.2.5
```

File: tests/system/provisioning/manager_agent/vars/configurations.yml [hunk 2]
```diff
@@ -22,7 +22,7 @@ agent3_id: "003"
 agent3_hostname: "wazuh-agent3"
 agent3_key: "3333333333333333333333333333333333333333333333333333333333333333"
 agent3_package: "https://{{package_repository}}.wazuh.com/{{repository}}/apt/pool/main/w/wazuh-agent/wazuh-agent_{{package_version}}-{{package_revision}}_amd64.deb"
-agent3_user: "root" #Only valid for agents verions=>4.2.5
+agent3_user: "root" #Only valid for agents versions=>4.2.5
```
Comment (davidjiglesias) [inline tests/system/provisioning/manager_agent/vars/configurations.yml:25]:
```suggestion
agent3_user: "root" #Only valid for agents version=>4.2.5
```

**Review by davidjiglesias — CHANGES_REQUESTED** [review]

File: tests/system/provisioning/manager_agent/vars/configurations.yml (hunk 2 above)
Comment (damarisg) [reply tests/system/provisioning/manager_agent/vars/configurations.yml]: Applied on https://github.com/wazuh/wazuh-qa/pull/4896/commits/47318b62d529af075bbbf52e63648cc6199aec62

**Pull request merged**